libafl/src/stages/mod.rs

```diff
@@ -498,3 +520,49 @@ pub mod pybind {
         Ok(())
     }
 }
+
+/// Trait for status tracking of stages which stash data to resume
+pub trait ResumableStageStatus<S> {
```

Comment:
Random question, could these stages, if implemented correctly, then be used as PushStage as well?

Reply:
Not quite sure. I've not done a whole lot with push stages.

Reply:
Basically instead of libafl running the fuzzing loop, libafl returns the next testcase.
So here, it would then "continue" after that

Guess it'd need some additional remodelling to be able to return instead of execute

Reply:
I think, after reviewing, we should be able to make push stage resumeable using the same strategy as e.g. If/IfElse/WhileStage.